projects/dex-ui/src/components/Liquidity/SlippagePanel.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React from "react";
+import styled from "styled-components";
+
+import arrowLeft from "/src/assets/images/arrow.svg";
+
+type SlippagePanelProps = {
+  slippageValue: number;
+  slippageValueChanged: (value: string) => void;
+  closeButtonClicked: () => void;
```

Comment:
nit on naming, this method will close the modal. IMO it should be named something like `closeModal`. using "button" or "clicked" in the name makes the assumption that it will be triggered by a button, and onClick, which neither has to be true per se